chains.yaml

```diff
@@ -59,15 +59,11 @@
   github-organization: axelarnetwork
   github-repo: tofnd
   language: rust
-  pre-build: |
-    apt install -y zlib1g-dev:${TARGETARCH}
-    TOOLCHAIN=$(cat rust-toolchain.toml | grep channel | awk '{print $3}' | tr -d '"')
-    rustup component add rust-src --toolchain ${TOOLCHAIN}-$(uname -m)-unknown-linux-gnu
-    git clone https://github.com/axelarnetwork/tofnd.git --recursive
```

Comment:
why was this removed?  This builds and runs for me

Reply:
We can use the build-target for this, don't need to do this manually